package.json

```diff
@@ -2,10 +2,15 @@
   "name": "taasa",
   "version": "0.0.12",
   "description": "Templates as a service",
+  "engines": {
+    "node": "8"
```

Comment:
Could we upgrade to Node v12 same as our other repositories? Then can we get away with just an .nvmrc?